src/plot_splines.jl

```diff
@@ -0,0 +1,110 @@
+using BSplineKit, Unfold
+"""
+    plot_splines(m::UnfoldModel; kwargs...)
+    plot_splines!(f::Union{GridPosition, GridLayout, Figure}, m::UnfoldModel; kwargs...)
+
+Shows two subfigures. First, baseses of splines. Second, density of splines.\\
+Dashed lines shows spline knots.
+
+## Arguments:
+
+- `f::Union{GridPosition, GridLayout, Figure}`
+    `Figure`, `GridLayout`, or `GridPosition` to draw the plot.
+- `m::UnfoldModel`\\
+    UnfoldModel with splines.
+$(_docstring(:splines))
+
+**Return Value:** `Figure` with splines and their density for basis functions.
+"""
+plot_splines(m::UnfoldModel; kwargs...) = plot_splines(Figure(), m; kwargs...)
+
+function plot_splines(f::Union{GridPosition,GridLayout,Figure}, m::UnfoldModel; kwargs...)
+    config = PlotConfig(:splines)
+    config_kwargs!(config; kwargs...)
+    ga = f[1, 1] = GridLayout()
+
+    terms = Unfold.formulas(m)[1].rhs.terms
+    spl_title = join(terms, " + ")
+
+    splFunction = Base.get_extension(Unfold, :UnfoldBSplineKitExt).splFunction
+    spl_ix = findall(isa.(terms, Unfold.AbstractSplineTerm))
+
+    spline_terms = [terms[i] for i in spl_ix]
+    subplot_id = 1
+    for spline_term in spline_terms
+        #basises = BSplineBasis(BSplineOrder(i.order), i.breakpoints)
+        #knots = BSplineKit.knots(basises)
+        x_range = range(
+            spline_term.breakpoints[1],
+            stop = spline_term.breakpoints[end],
+            length = 100,
+        )
+        basis_set = splFunction(x_range, spline_term)
+
+        ylabel_first = "Spline value"
+        if subplot_id > 1
+            config_kwargs!(config; axis = (; ylabelvisible = false))
+            ylabel_first = ""
+        end
+        a1 = Axis(
+            ga[1, subplot_id];
+            title = string(spline_term),
+            ylabel = ylabel_first,
+            xlabelvisible = false,
+            xticklabelsvisible = false,
+        )
+        series!(
+            x_range,
+            basis_set',
+            color = resample_cmap(config.visual.colormap, size(basis_set')[1]),
+        ) #contionus colormap used
+        vlines!(
+            spline_term.breakpoints;
+            ymin = extrema(basis_set')[1],
+            ymax = extrema(basis_set')[2],
+            linestyle = :dash,
+        )
+        #scatter!(spline_term.breakpoints, [0,0,0,0,0,0,0,0]; markersize = 10, strokecolor = :tomato, strokewidth = 3)
+        a2 = Axis(
+            ga[2, subplot_id];
+            xlabel = string(spline_term.term.sym),
+            xautolimitmargin = (0, 0),
+            config.axis...,
+        )
+        density!(
+            Unfold.events(designmatrix(m))[1][:, spline_term.term.sym];
+            color = :transparent,
+            strokecolor = :black,
+            strokewidth = 1,
+        )
+        linkxaxes!(a1, a2)
+        subplot_id = subplot_id + 1
+    end
+
+    supertitle =
```

Comment:
do we have a "title" or similar in the plotconfig? via axis or so? In any case, could be good to expose them for modification?